dialect/sql/builder.go

```diff
@@ -908,6 +914,12 @@ func NEQ(col string, value interface{}) *Predicate {
 	return P().NEQ(col, value)
 }
 
+// EQCol returns a "=" predicate for "col<>col", omit Raw.
+// example: NEQ(`created_at`, Raw(`updated_at`)) <=> NEQCol(`created_at`, `updated_at`)
+func NEQCol(col string, col2 string) *Predicate {
+	return P().NEQ(col, Raw(col2))
+}
+
```

Comment:
This 2 functions are not really needed by the default codegen, and can also be written as follows:

```go
p := sql.P(func(b *sql.Builder) {
	b.Ident("a").WriteOp(sql.OpEQ).Ident("b") // OpEQ can be replaced with any other operator.
})
```
Let's remove them.

Reply:
I might just want to save two lines of code
will remove *EQCol